### ISSUE TITLE
Run all Dusk/Browser tests via `php artisan test`

### ISSUE THREAD

**Issue by susanBuck** [laravel/dusk#896]
If I run `php artisan dusk` it finds/runs all my Dusk/Browser tests, reporting back success/failure using standard phpunit output:

![image](https://user-images.githubusercontent.com/237711/115779900-8824f280-a386-11eb-89b3-77aebcd783e2.png)

If I run **an individual Dusk** test via `php artisan test`, the output is much nicer with clearer formatting/color-coding and indication of the exact line where the problem is occuring.

![image](https://user-images.githubusercontent.com/237711/115780029-ae4a9280-a386-11eb-835d-699b1a4fb371.png)

Seeing this, I want to be able to run all my Dusk tests this way. Referencing the docs ([Running tests](https://laravel.com/docs/8.x/testing#running-tests)), I think should be able to do this via `php artisan --testsuite=Browser`, but that yields no tests:

![image](https://user-images.githubusercontent.com/237711/115780381-17caa100-a387-11eb-9f00-154426a72348.png)

**In summary: Is it possible to run all Dusk tests using the _php artisan test_ command with the pretty output?**

**Comment by driesvints** [comment]
Resubmit this with a filled out issue template.

**Comment by susanBuck** [comment]
Done, thanks. Wasn't sure if this was an issue or a feature request, so I went with the latter.
